src/assets/javascript/components/clearAllButton.js

```diff
@@ -0,0 +1,6 @@
+function clearAllButton(textContent) {
+
+	return `<button class="js-btn-iniciais js-btn-ClearAll">${textContent}</button>`;
```

Comment:
You don't need this

Reply:
do you think it's better to separate all the buttons with different functions on different functions? i mean,on different files
or
i just separate the necessary?

Reply:
what is the benefit to create many button component with almost the same code?

Reply:
@eerison 
i'm using **if else** for the pattern buttons, because they do the same thing, **_print_**
and for the other ones that have different functions like delete,clear, show the result, change signals i'm using **switch case**.
do you think i'ts better now ? or it's more complicated?